### PR TITLE
Info page

### PR DESCRIPTION
**Pull request by MaksimkaSota**
1. Added InfoPage component;
2. Added more contacts and information about the page.

### CODE REVIEW

File: src/Components/Common/Buttons/ChartSelectionButton/ChartSelectionButton.tsx [hunk 1]
```diff
@@ -36,7 +38,7 @@ export const ChartSelectionButton: FC<PropsType> = ({
     // @ts-ignore
     <CustomTag {...props}>
       <img src={src} alt={text} />
-      <p className={classNameText}>{text}</p>
+      {!isInfoPage && <p className={classNameText}>{text}</p>}
```
Comment (ZhekaVasil) [inline src/Components/Common/Buttons/ChartSelectionButton/ChartSelectionButton.tsx]: I think it is not a good name for the prop, the "button" component should not know about your pages at all
It makes sense to rename it to smth like "showLabel, showText, showName" or "hideName", so the prop should be "text" specific in your case

File: src/Components/Common/Table/Table.module.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    width: 100%;
+    padding: $indentInputAndButton 0 0;
+    box-sizing: border-box;
+    .table {
+      font-size: 0.55rem;
+    }
+  }
+  @include respond($mobile) {
+    .table {
+      font-size: 0.6rem;
```
Comment (ZhekaVasil) [inline src/Components/Common/Table/Table.module.scss]: BTW, do you have constants for font sizes? I would recommend to create them and have limited set of sizes so you could easily refer to them in scss file

File: src/Components/Common/Table/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+export const Table: FC<PropsType> = ({ tableValues }): ReactElement => {
+  return (
+    <div className={classes.tableContainer}>
+      <table className={classes.table}>
+        <tbody>
+          {tableValues.map(
+            (rowValue: (string | number)[], rowIndex: number): ReactElement => (
+              <tr className={classes.tr} key={rowIndex}>
+                {rowValue.map((columnValue: string | number, columnIndex: number): ReactElement => {
+                  return rowIndex === 0 || columnIndex === 0 ? (
```
Comment (ZhekaVasil) [inline src/Components/Common/Table/Table.tsx]:
```suggestion
                  return !rowIndex || !columnIndex ? (
```

File: src/Components/Main/Pages/ExamplesPage/ExampleChart/ExampleChart.tsx [hunk 4]
```diff
@@ -12,17 +13,20 @@ type PropsType = {
...
-  ({ isFetchingAddress, address, addressError, options, setNewOptions }): ReactElement => {
+  ({ isFetchingAddress, address, addressError, options, setNewOptions, isInfoPage, tableValues }): ReactElement => {
     return (
       <div className={classes.exampleChart}>
         <div className={classes.wrapper}>
           <div className={classes.imageContainer}>
             <Chart isFetchingAddress={isFetchingAddress} address={address} addressError={addressError} />
           </div>
         </div>
+        {isInfoPage && tableValues && <Table tableValues={tableValues} />}
```
Comment (ZhekaVasil) [inline src/Components/Main/Pages/ExamplesPage/ExampleChart/ExampleChart.tsx]: The same here, isInfoPage is not the best naming here, you could try - "showTable", "hideTable" or smth else

File: src/Components/Main/Pages/ExamplesPage/ExampleCharts/ExampleCharts.tsx [hunk 5]
```diff
@@ -36,30 +41,41 @@ export const ExampleCharts: FC<PropsType> = ({
...
+  isInfoPage,
 }): ReactElement => {
+  const exampleFirstTableValues = getTableValues('Численность по годам', exampleFirstInitialValue);
+  const exampleSecondTableValues = getTableValues('Заработная плата по месяцам', exampleSecondInitialValue);
+  const exampleThirdTableValues = getTableValues('Инфляция по годам', exampleThirdInitialValue);
+
   return (
     <div className={classes.chartResult}>
-      {!isMainPage && <h3 className={classes.miniTitle}>График</h3>}
+      {!isMainPage && !isInfoPage && <h3 className={classes.miniTitle}>График</h3>}
```
Comment (ZhekaVasil) [inline src/Components/Main/Pages/ExamplesPage/ExampleCharts/ExampleCharts.tsx]: Same here for isInfoPage  prop

File: src/Components/Common/Buttons/ChartSelectionButton/ChartSelectionButton.tsx (hunk 1 above)
Comment (MaksimkaSota) [reply src/Components/Common/Buttons/ChartSelectionButton/ChartSelectionButton.tsx]: Done!

File: src/Components/Main/Pages/ExamplesPage/ExampleChart/ExampleChart.tsx (hunk 4 above)
Comment (MaksimkaSota) [reply src/Components/Main/Pages/ExamplesPage/ExampleChart/ExampleChart.tsx]: Done!

File: src/Components/Main/Pages/ExamplesPage/ExampleCharts/ExampleCharts.tsx (hunk 5 above)
Comment (MaksimkaSota) [reply src/Components/Main/Pages/ExamplesPage/ExampleCharts/ExampleCharts.tsx]: Done!

File: src/Components/Common/Table/Table.tsx (hunk 3 above)
Comment (MaksimkaSota) [reply src/Components/Common/Table/Table.tsx]: Done!

**Review by ZhekaVasil — DISMISSED** [review]

File: src/Components/Common/Table/Table.module.scss (hunk 2 above)
Comment (MaksimkaSota) [reply src/Components/Common/Table/Table.module.scss]: Done!

**Review by ZhekaVasil — APPROVED** [review]

**Pull request merged**